WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -877,6 +904,19 @@ private extension NotificationsViewController {
             resetApplicationBadge()
         }
     }
+
+    func updateMarkAllAsReadTintColor() {
```

Comment:
This is a bit nitpicky, but this method updates the button as well as tint color, so maybe the name can be a little less specific? Like `updateMarkAllAsReadButton` maybe?